docs/sources/dashboards/_index.md

```diff
@@ -32,7 +32,7 @@ cards:
       description: Add variables to metric queries and panel titles to create interactive and dynamic dashboards.
       height: 24
     - title: Public dashboards
-      href: ./dashboard-public/
+      href: ./share-dashboards-panels/shared-dashboards/
```

Comment:
why `panels` naming?

Reply:
The page has moved so I have to update the pathway